package.json

```diff
@@ -158,29 +158,29 @@
     "enzyme-to-json": "^3.4.3",
     "eslint": "^6.6.0",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.5.0",
+    "eslint-config-prettier": "^6.6.0",
     "eslint-import-resolver-alias": "^1.1.2",
     "eslint-plugin-cypress": "^2.7.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-json": "^2.0.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-react": "^7.16.0",
-    "eslint-plugin-react-hooks": "^2.2.0",
+    "eslint-plugin-react-hooks": "^2.3.0",
     "history": "^4.10.1",
     "husky": "^3.0.9",
     "inspectpack": "^4.2.2",
     "jest": "^24.9.0",
     "jest-fetch-mock": "^2.1.2",
-    "jest-styled-components": "^6.3.3",
+    "jest-styled-components": "^6.3.4",
     "jsdom": "^15.2.1",
-    "lint-staged": "^9.4.2",
+    "lint-staged": "^9.4.3",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.19.1",
     "prop-types": "^15.7.2",
     "start-server-webpack-plugin": "^2.2.5",
     "storybook-chromatic": "^3.1.0",
-    "stylelint": "^11.1.1",
+    "stylelint": "^12.0.0",
```

Comment:
This is a major change, worth checking the changelog to see if we expect any differences

Reply:
From reading the [changelog](https://github.com/stylelint/stylelint/releases/tag/12.0.0) it looks like the changes do not impact us.